include/gridtools/common/int_vector.hpp

```diff
@@ -0,0 +1,161 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2014-2021, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * @file
+ *
+ * Implements operations on `int_vector`s, which are hymaps of integers (integral types or integral_constants)
+ *
+ * A type `T` is `int_vector` if
+ *  - `is_hymap<T>`
+ *  - all value types `V` are std::is_integral<V> or is_gr_integral_constant<V>
+ *  - `std::is_trivially_copy_constructible<T>` // TODO isn't that trivially satisfied by the previous conditions?
+ */
+
+#pragma once
+
+#include <type_traits>
+
+#include "../meta.hpp"
+#include "defs.hpp"
+#include "gridtools/meta/debug.hpp"
```

Comment:
not used

Reply:
Thanks for checking the includes! clangd auto includes this everytime I complete GT_META_PRINT_TYPE...